src/fork_gc.c

```diff
@@ -536,13 +536,27 @@ static void FGC_childCollectMissingDocs(ForkGC *gc, RedisSearchCtx *sctx) {
   FGC_sendTerminator(gc);
 }
 
+static void FGC_childCollectExistingDocs(ForkGC *gc, RedisSearchCtx *sctx) {
```

Comment:
Can we get code coverage for this function in our tests?

Reply:
This is the fork code, so no